### DISCUSSION TITLE
RC 96 release date?

### DISCUSSION THREAD

**Discussion by nichols-green** [directus/directus#8613] [Archive]
The scope of the RC96 milestone seems to endlessly increase. There are some critical fixes related to MSSQL in RC96 that my team and I are waiting for. We may be the only ones concerned with this, but we thought we would at least bring it up. The time between releases has historically been very fast, but this release seems to be the exception to that. Any idea as to when 96 might make it to an actual release?

**Comment by joselcvarela** [comment]
Hello @nichols-green 

We do not have specific interval between releases. You can check here the history: https://github.com/directus/directus/tags

We are available for expedited fixes. Here are the requisites: https://docs.directus.io/getting-started/support/#expedited-fixes

Also, you can track the progress of the release: https://github.com/directus/directus/milestone/79

**Reply by nichols-green** [reply]
Appreciate the response and links. I'm aware of those already but others may find it valuable. I have been tracking the progress of RC96 and that's actually my main point. The progress reached 96%, and at the time of this writing is back down to 87% as tickets are continually added to the milestone. This makes milestone tracking far less valuable as the scope changes. What determines whether a milestone is "full" or not? Is there any indication for whether a milestone is open or full? 

Not trying to be a drag, I love the work you all are doing and will continue to contribute where possible, just trying to draw some attention to a problem I am experiencing while following the project.

**Reply by rijkvanzanten** [reply]
We've historically released a new RC every 5-15 days. The main blocker now for rc 96 is https://github.com/directus/directus/pull/8570. Once that's in (which should be in a couple hours now) we'll release 96. In the meantime, we'll add any smaller ticket or issue that we might be able to "sneak in" as a team while that main feature is being wrapped up.

> What determines whether a milestone is "full" or not?

In these RC releases, there have often been one or more "big" changes that have to be finished and "sit" in main for a minute to ensure we didn't break things totally. The determination of when a release is ready to be pushed out is mostly a question of stability on "main", and feature goals we're aiming to hit in particular versions.

**Reply by nichols-green** [reply]
That's a much appreciated explanation, thank you. Is there any way in GH to indicate the main or big ticket item for the RC milestone? That would be very handy IMO.

**Reply by rijkvanzanten** [reply]
Good one! We could look into a label like "blocker" or something like that 🤔 

Alternatively, it might be even more useful if we start releasing on a fixed day/time rather than based on features to include

**Reply by nichols-green** [reply]
I love the label idea! I don't want to shake up the flow if that flow is working. Both options work, so I feel like my objective is met 😄

**Reply by rijkvanzanten** [reply]
@nichols-green I'll do you one better: we'll start releasing on a fixed schedule. Starting today, we'll release every thursday morning (nyc local time) 🙂
